server/infrastructure/supabase/mappers/StudentMapper.ts

```
@@ -0,0 +1,67 @@
+      admission_year: academic.getAdmissionYear(),
+      semester_enrolled: academic.getSemester(),
+      is_transfer_student: academic.getIsTransfer(),
+      status: academic.getStatus() as DatabaseStudentStatus,
+      grade_level: academic.getGradeLevel(),
+      completed_semesters: academic.getCompletedSemesters(),
+      target_gpa: domain.getTargetGpa() ?? null,
+    };
+  }
+}
+
+export class StudentMappingError extends Error {
```

Biome (1.9.4)

[error] Avoid classes that contain only static members.

Prefer using simple functions instead of classes with only static members.

(lint/complexity/noStaticOnlyClass)